app/controllers/orders_controller.rb

```diff
@@ -8,49 +8,41 @@ def index
 
   def create
     date = Date.today
-    user_id = @current_user.is_owner || @current_user.is_clerk ? 4 : current_user.id
-    menu_item_ids = params[:menu_item_ids]
+    user_id = current_user.id
 
-    if !menu_item_ids
-      flash[:error] = "Please select at least one item to place an order."
-      redirect_to menus_path
-      return
+    cart = Cart.find_by user_id: user_id
```

Comment:
**Suggestion:** Avoid querying the database multiple times for the same data. You can store the result of `Cart.find_by user_id: user_id` in a variable and reuse it.
```suggestion
    cart = Cart.find_by user_id: user_id
    ...
    total: cart.total,
    ...
    cart_items = cart.cart_items
    cart_items.each do |item|
      ...
    end
    cart_items.delete_all
    cart.update(
      total: 0.0,
    )
```